build/common.props

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="..\version.props" />
```

Comment:
Yeah, works in VS and command line. Something else must be going on since the import would fail if it couldn't find the file.